tests/cpp/test_gpu3.cpp

```diff
@@ -31,6 +31,8 @@
 #include <kernel_ir.h>
 #include <kernel_ir_dispatch.h>
 #include <ops/all_ops.h>
+#include <preseg_passes/mark_aliases_prepare.h>
+#include <preseg_passes/optimization_pass.h>
```

Comment:
nitpick: Don't see any reason to add these headers.

Reply:
Right, this is a remnant from previous changes. Thanks for catching!